### PR TITLE
Added modern property to SkyDocsComponentInfo

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Made to match the supportal service:
https://blackbaud.visualstudio.com/Products/_git/sky-skysp-svc?path=%2Fsrc%2FBlackbaud.SkyUX.SupportalService%2FModels%2FDocsComponentInfo.cs&version=GBmaster&line=14&lineEnd=15&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-property](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-property)

File: src/app/public/modules/shared/docs-tools-component-info.ts [hunk 1]
```diff
@@ -9,4 +9,5 @@ export interface SkyDocsComponentInfo {
   summary?: string;
   thumbnail?: string;
   url?: string;
+  modern?: string;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/shared/docs-tools-component-info.ts:12]: Isn't this a `boolean`?

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/shared/docs-tools-component-info.ts:12]: Also, alphabetical order, please.

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/shared/docs-tools-component-info.ts]: yes. fixed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-property](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-property)

**Pull request merged**